symbolic-debuginfo/src/macho.rs

```diff
@@ -206,6 +257,7 @@ impl<'d> MachObject<'d> {
             symbols: self.macho.symbols(),
             sections,
             vmaddr: self.load_address(),
+            symbolmap: self.bcsymbolmap.clone(), // TODO(flub): Don't clone
```

Comment:
We'll have to keep doing this because this iterator does not depend on the lifetime of the object itself.  The symbol map is now only a vector of pointers, so it's not so bad, but it is kind of sad.